src/SFA.DAS.Authorization/DependencyResolution/Microsoft/ServiceCollectionExtensions.cs

```diff
@@ -22,6 +22,7 @@ public static IServiceCollection AddAuthorization(this IServiceCollection servic
                 .AddMemoryCache()
                 .AddScoped<IAuthorizationContextProvider>(p => new AuthorizationContextCache(p.GetService<T>()))
                 .AddScoped<IAuthorizationService, AuthorizationService>()
+                .AddScoped<IAuthorizationService, AuthorizationServiceWithDefaultHandler>()
```

Comment:
The IDefaultAuthorizationHandler should also be set to the AuthorizationServiceWithDefaultHandler here.  It's only set up in the AuthorizationRegistry for older clients.   You can use the Scrutor nuget package to provide the decorate method.

Also should also add a way for the caller to set up the default handler at this point.  e.g.  by passing in an AuthorizationOptions class and checking if a DefaultHandler option has been set and therefore use that one instead of the default implementation.